evidencecore/cryptolib/readkey_test.go

```diff
@@ -0,0 +1,32 @@
+package cryptolib
+
+import (
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+func TestReadKey(t *testing.T) {
+	files, err := os.ReadDir("test-data")
+	assert.Nil(t, err)
+	assert.Equal(t, 14, len(files))
+	var keyFiles []os.DirEntry
+	keysToValidate := []string{"ecdsa-test-key-pem", "ed25519-test-key-pem", "rsa-test-key"}
+	for _, file := range files {
+		for _, key := range keysToValidate {
+			if file.Name() == key {
+				keyFiles = append(keyFiles, file)
+			}
+		}
+
+	}
+	assert.Equal(t, 3, len(keyFiles))
+
+	for _, file := range keyFiles {
+		keyFile, err := os.ReadFile("test-data" + "/" + file.Name())
```

Comment:
Please avoid using raw `/` because it may not work on Windows
```suggestion
		keyFile, err := os.ReadFile(filepath.Join("testdata", file.Name()))
```